compatibility_server/pip_checker.py

```diff
@@ -354,46 +355,79 @@ def _build_command(self, subcommands: List[str]):
         """Build pip commands."""
         return self._pip_command + subcommands
 
+    def _clone_repo(self, container: docker.models.containers.Container):
+        """Shallow clone the google-cloud-python repository."""
+        # Create a temp directory
+        _, directory = self._run_command(
+            container,
+            ['mktemp', '-d'],
+            stderr = True,
```

Comment:
I don't think that you want stderr because it could interfere with the directory name.

Reply:
Done.